src/magicbox/ResultPreviewsGrid.ts

```diff
@@ -0,0 +1,398 @@
+import { $$, Dom } from '../utils/Dom';
+import _ = require('underscore');
+import { Assert } from '../misc/Assert';
+import { defaults } from 'underscore';
```

Comment:
You already imported underscore as _ on line 2, maybe it would be better to deconstruct every underscore method that you need and remove line 2?

Reply:
Good point, I used intellisense to automatically import everything I used and didn't think twice about checking the imports it generated. I'm fixing it.